education/cryptocurrency/economics.md

```diff
@@ -13,7 +13,7 @@ Let's use an example to show what makes a cryptocurrency valuable.
 **The US Dollar** is a currency used to buy things in the United States. A dollar is a piece of paper with some numbers on it. What makes it worth anything? A few decades ago, the US dollar was backed by gold, meaning that for every dollar that existed, there was a little bit of gold being held in a safe somewhere. If you wanted, you could trade your dollar for that gold. But the dollar hasn't been backed by gold since [1934](https://www.federalreserve.gov/faqs/currency_12770.htm). If we went back in time to 1933, we could say that the dollar had value because it could be traded for gold, but now that isn't really true...the dollar has value now because everyone agrees that it has value.
 
 #### Are cryptocurrencies a limited resource?
-For most cryptocurrencies, there is a limited number of "coins" that are available at any given time. You can't just "create" a bitcoin out of nowhere and add it to your wallet. Coins are created by "miners" who are kinda like a government's mint. Because miners can only create a limited number of coins in most cryptocurrencies, the coins are a limited resource.
+For most cryptocurrencies, there is a limited number of "coins" that are available at any given time. You can't just "create" a bitcoin out of nowhere and add it to your wallet. Coins are created by "miners" who are kind of like a government's mint. Miners can only create a limited number of coins in most cryptocurrencies, so the coins are a limited resource.
```

Comment:
I like that. I think "the way a government prints new money" might flow better but either one is cool with me